build.sbt

```diff
@@ -1151,19 +1206,13 @@ lazy val `ydoc-server` = project
   .configs(Test)
   .settings(
     frgaalJavaCompilerSetting,
+    javaModuleName := "org.enso.ydoc",
     crossPaths := false,
     autoScalaLibrary := false,
-    Compile / run / fork := true,
     Test / fork := true,
-    run / connectInput := true,
-    Compile / run := (Compile / run)
-      .dependsOn(
-        Def.task {
-          import scala.sys.process._
-          "npm --workspace=enso-gui2 run build-ydoc-server-polyglot" ! streams.value.log
-        }
-      )
-      .evaluated,
+    assembly := assembly
```

Comment:
At the end, the `assembly` task along with all its setting (`assembly/assemblyExcludedJars`, etc.) should not be needed. To package `ydoc-server` and run it, it should be enough to do `ydoc-server/packageBin` which will just compile classes inside that project and package it in a JAR. So at the end `ydoc-server.jar` will have just classes and resources from `ydoc-server` and nothing else.